backends/cuda/ceed-cuda-vector.c

```diff
@@ -348,6 +348,55 @@ static int CeedVectorRestoreArray_Cuda(const CeedVector vec) {
   return 0;
 }
 
+//------------------------------------------------------------------------------
+// Restore an array obtained using CeedVectorGetArray()
+//------------------------------------------------------------------------------
+static int CeedVectorNorm_Cuda(CeedVector vec, CeedNormType type,
+                               CeedScalar *norm) {
+  int ierr;
+  Ceed ceed;
+  ierr = CeedVectorGetCeed(vec, &ceed); CeedChk(ierr);
+  Ceed_Cuda *ceed_Cuda;
+  ierr = CeedGetData(ceed, (void *) &ceed_Cuda); CeedChk(ierr);
+  CeedVector_Cuda *data;
+  ierr = CeedVectorGetData(vec, (void *)&data); CeedChk(ierr);
+  CeedInt length;
+  ierr = CeedVectorGetLength(vec, &length); CeedChk(ierr);
+  if (!ceed_Cuda->cublasHandle) {
+    ierr = cublasCreate(&ceed_Cuda->cublasHandle); CeedChk_Cublas(ceed, ierr);
+  }
+  cublasHandle_t handle = ceed_Cuda->cublasHandle;
```

Comment:
Could we capture this pattern?
```suggestion
  cublasHandle_t handle;
  ierr = CeedCudaGetCublasHandle(ceed, &handle); CeedChk(ierr);
```

Reply:
argeed